ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -87,8 +87,15 @@ requests
 ${local.skip_on_weekends}
 | where toint(resultCode) between (200 .. 299) and timestamp >= ago(5m)
 | sort by timestamp asc
-| summarize alert = iff((todouble(sumif(1, success == false)) * 100 / todouble(count()) >= ${var.http_2xx_failure_rate_threshold}), 1, 0)
-| where alert == 1
+| summarize alert = iff((todouble(sumif(1, success == false)) * 100 / todouble(count()) >= ${var.http_2xx_failure_rate_threshold} and sumif(1, success == false) > ${var.http_2xx_failed_threshold}), sumif(1, success == false), 0)
+| where alert > 0
+//
+// You can use the following query to see all the requests that triggered this alert.
```

Comment:
Will this comment appear in the PagerDuty link that is generated when the alert fires? (I don't know too much about that integration, so I'm not sure if this is all it takes to update the link or if the comment will just be stored here in the code)

Reply:
When you click the link in pagerduty into the azure portal, this comment will appear in the query box. It'll appear like I have pasted in the PR description, with the percentage value, threshold, and timestamps all filled in.

Please let me know if I have fully answered your question!

Reply:
You did! Thank you :)